modules/core/src/test/scala/com/azavea/stac4s/CatalogSpec.scala

```diff
@@ -0,0 +1,436 @@
+          description = "STAC for Landsat data",
+          links = List(
+            StacLink(
+              href = "https://landsat-stac.s3.amazonaws.com/catalog.json",
+              rel = StacLinkType.Self,
+              _type = None,
+              title = None,
+              // should it be an optional thing?
+              labelExtAssets = Nil
+            ),
+            StacLink(
+              href = "./catalog.json",
+              rel = StacLinkType.StacRoot,
+              _type = None,
+              title = None,
+              // should it be an optional thing?
+              labelExtAssets = Nil
+            ),
+            StacLink(
+              href = "landsat-8-l1/catalog.json",
+              rel = StacLinkType.Child,
+              _type = None,
+              title = None,
+              // should it be an optional thing?
+              labelExtAssets = Nil
+            )
+          )
+        )
+
+      root.asJson.deepDropNullValues shouldBe getJson("/catalogs/landsat8/catalog-v1.json")
+      root.asJson.as[StacCatalog].valueOr(throw _) shouldBe root
+    }
+
+    it("Create LC8 Collection") {
+      val collection = StacCollection(
+        stacVersion = "0.9.0",
+        stacExtensions = List("eo", "view", "https://example.com/stac/landsat-extension/1.0/schema.json"),
+        id = "landsat-8-l1",
+        title = Some("Landsat 8 L1"),
+        description =
+          "Landat 8 imagery radiometrically calibrated and orthorectified using gound points and Digital Elevation Model (DEM) data to correct relief displacement.",
+        keywords = List("landsat", "earth observation", "usgs"),
+        license = Proprietary(), //SPDX("PDDL-1.0"), // PDDL-1.0
+        providers = List(
+          StacProvider(
+            name = "Development Seed",
+            description = None,
+            roles = List(Processor),
+            url = Some("https://github.com/sat-utils/sat-api")
+          )
+        ),
+        extent = StacExtent(
+          spatial = SpatialExtent(List(TwoDimBbox(-180, -90, 180, 90))),
+          temporal = Interval(
+            List(TemporalExtent(Instant.parse("2013-06-01T00:56:49.001Z"), Instant.parse("2020-01-01T00:56:49.001Z")))
+          )
+        ),
+        // properties can be anything
+        // it is a part where extensions can be
+        // at least EO, Label and potentially the layer extension
+        properties = JsonObject.fromMap(
+          Map(
+            "collection"         -> "landsat-8-l1".asJson,
+            "eo:gsd"             -> 15.asJson,
+            "eo:platform"        -> "landsat-8".asJson,
+            "eo:instrument"      -> "OLI_TIRS".asJson,
+            "view:off_nadir"     -> 0.asJson,
+            "view:sun_azimuth"   -> 149.01607154.asJson,
+            "view:sun_elevation" -> 59.21424700.asJson,
+            "view:azimuth"       -> 0.asJson,
+            "eo:bands" -> List(
+              JsonObject.fromMap(
```

Comment:
Do you need to wrap the maps in `JsonObject.fromMap`? I'd think a `List[Map[String, Json]]` would be serializable / fine on its own

Reply:
Oops I pushed changes, but they are not visible in this PR for some reason ._.; 

https://github.com/pomadchin/stac4s/tree/feature/lc8-specs